docker/prod/site.conf

```diff
@@ -51,9 +51,5 @@ server {
         fastcgi_param SERVER_NAME $http_x_original_host;
     }
 
-    # handle short URLs - "/Foo" instead of "/wiki/Foo"
-    # but do not break known entry points (SUS-5271)
-    if ($request_filename !~ "(load|wikia|api).php") {
-        rewrite "^/(?:[a-z]{2,3}(?:-[a-z-]{2,12})?/)?(.*)" /index.php?title=$1 break;
-    }
+    error_page 404 = /redirect-canonical.php;
```

Comment:
Our of curiosity, how does this behave with language path wikis?

Reply:
Broken. Same with production apaches 😄 

@themech says there is a PLATFORM ticket for that